src/tasks/Common/Utils.cs

```diff
@@ -68,7 +68,7 @@ public static string GetEmbeddedResource(string file)
             {
                 if (!silent)
                 {
-                    LogError(e.Data);
+                    LogInfo(e.Data, outputMessageImportance);
```

Comment:
This method is used in many other places too. Maybe we should add a method param like `logStdErrAsError` or something, defaulting to true?

Reply:
It seems what's being written to stderr from the aot compiler are warnings more than anything.  Would it make better sense to just log a warning here?

Reply:
Yes I'd prefer warning instead of info.